roxctl/scanner/uploaddb/uploaddb_test.go

```diff
@@ -1,18 +1,101 @@
 package uploaddb
 
 import (
+	"bytes"
+	"fmt"
 	"io/fs"
+	"net/http"
+	"net/http/httptest"
+	"os"
 	"testing"
 
+	"github.com/stackrox/rox/pkg/utils"
+	"github.com/stackrox/rox/roxctl/common/environment"
+	"github.com/stackrox/rox/roxctl/common/flags"
+	"github.com/stackrox/rox/roxctl/common/printer"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-func TestScannerUploadDbCommandFail(t *testing.T) {
-	cmdNoFile := scannerUploadDbCommand{filename: "non-existing-filename"}
+func executeUpdateDbCommand(serverURL string) (*bytes.Buffer, *bytes.Buffer, error) {
+	tmpFile, errTempFile := os.CreateTemp("", "*.zip")
+	if errTempFile != nil {
+		return nil, nil, errTempFile
+	}
+	defer utils.IgnoreError(func() error {
+		return os.Remove(tmpFile.Name())
+	})
 
-	actualErr := cmdNoFile.uploadDd()
+	testIO, _, stdOut, stdErr := environment.TestIO()
+	env := environment.NewCLIEnvironment(testIO, printer.DefaultColorPrinter())
 
-	require.Error(t, actualErr)
-	assert.ErrorIs(t, actualErr, fs.ErrNotExist)
+	cmd := Command(env)
+	flags.AddTimeout(cmd)
+	flags.AddConnectionFlags(cmd)
+	flags.AddPassword(cmd)
+
+	cmdArgs := []string{"--insecure-skip-tls-verify", "--insecure", "--endpoint", serverURL, "--password", "test"}
```

Comment:
Maybe add a comment that explains why this is needed and a todo to fix it

Reply:
✅